migration/migrator/migrations/course/20211117160515_add_late_day_triggers.py

```diff
@@ -0,0 +1,376 @@
+"""Migration for the Submitty system."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+
+    # Drop triggers
+    database.execute("DROP TRIGGER IF EXISTS gradeable_version_change ON electronic_gradeable_version;")
+    database.execute("DROP TRIGGER IF EXISTS late_days_allowed_change ON late_days;")
+    database.execute("DROP TRIGGER IF EXISTS late_day_extension_change ON late_day_exceptions;")
+    database.execute("DROP TRIGGER IF EXISTS electronic_gradeable_change ON electronic_gradeable;")
+    database.execute("DROP TRIGGER IF EXISTS gradeable_delete ON gradeable;")
+
+    # Create functions
+    database.execute("""
+    CREATE OR REPLACE FUNCTION grab_late_day_gradeables_for_user(user_id text) RETURNS SETOF late_day_cache
+        LANGUAGE plpgsql
+        AS $$
+    #variable_conflict use_variable
+    DECLARE
+    latestDate timestamp without time zone ;
```

Comment:
Is there a particular reason why this doesn't have a timezone?

Reply:
good point